lite/kernels/nnadapter/bridges/conv_op.cc

```diff
@@ -245,9 +267,25 @@ int ConvConverter(void* ctx, OpLite* op, KernelBase* kernel) {
       fuse_code_operand,
       dilation_width_operand,
       dilation_height_operand};
-  std::vector<NNAdapterOperand*> output_operands = {output_operand};
+  std::vector<NNAdapterOperand*> output_operands = {immediate_operand};
   auto conv2d_operation = converter->AddOperation(NNADAPTER_CONV_2D);
   converter->SetOperation(conv2d_operation, &input_operands, &output_operands);
+
+  // Activation operation without fusion
+  if (!conv_with_act_fusion) {
```

Comment:
直接在这里实现如下代码即可
 if (!act_type.empty()) {
    std::vector<NNAdapterOperand*> activation_input_operands{output_operand};
    // TODO(csy0225): Quant scene needs to save the conv output's scale, because the act op's input needs it.
    if (has_output_scale) {
      LOG(WARNING) << "NNadapter conv + act fusion op which splits into two operators doesn't support quant scene!";
      return FAILED;
      }
      output_operand = converter->AddFloat32VariableOperand(output_dims);
    }
    std::vector<NNAdapterOperand*> activation_output_operands{output_operand};
    if (act_type == "leaky_relu") {
      activation_operation = converter->AddOperation(NNADAPTER_LEAKY_RELU);
      auto alpha = op_info->GetAttr<float>("leaky_relu_alpha");
      auto alpha_operand = converter->AddFloat32ConstantOperand(alpha);
      activation_input_operands.push_back(alpha_operand);
   } else if (act_type == "sigmoid") {
    ....
   } else {
     LOG(FATAL) <<  "xxxx";
  }
}